contracts/GolemNetworkToken2.sol

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 Golem Factory
+// Licensed under the GNU General Public License v3. See the LICENSE file.
+
+pragma solidity ^0.4.19;
+
+import "./ExtendedToken.sol";
```

Comment:
I dunno if ExtendedToken is worth extracting after all, we could just move its methods into this new Token.

Reply:
I kinda agree, but might be easier to review this if generic implementation (like batchTransfer()) are separated.

Reply:
We'd also remove diamond inheritance which might be problematic. I honestly don't know how it works in Solidity.

Reply:
We will not. It would be `GNT2 is TokenProxy, StandardToken`.

Reply:
contract TokenProxy is ERC20Basic
would it be ok?

Reply:
I will do it. Now or in the next PR?